modules/db/models/community.js

```diff
@@ -0,0 +1,163 @@
+const timestamps = require('mongoose-time')
+const BigNumber = require('bignumber.js')
+
+module.exports = (db) => {
+  const Schema = db.mongoose.Schema
+
+  const setDecimal128 = (bignum) => {
+    return db.mongoose.Types.Decimal128.fromString(bignum.toString())
+  }
+
+  const getDecimal128 = (decimal) => {
+    const val = decimal ? decimal.toString() : 0
+    return new BigNumber(val)
+  }
+
+  const BalanceSchema = new Schema({
+    currency: {type: Schema.Types.ObjectId, ref: 'Currency'},
+    blockchainBalance: {type: db.mongoose.Schema.Types.Decimal128, set: setDecimal128, get: getDecimal128},
+    offchainBalance: {type: db.mongoose.Schema.Types.Decimal128, set: setDecimal128, get: getDecimal128}
+  }).plugin(timestamps())
+
+  const CommunitySchema = new Schema({
+    managerWalletAddress: {type: String},
+    usersWalletAddress: {type: String},
+    mnemonic: {type: String},
+    defaultCurrency: {type: Schema.Types.ObjectId, ref: 'Currency'},
+    managerBalance: {type: BalanceSchema},
```

Comment:
Balances should be Array of balances, where manager balance is the first balance of that array,
see [comment](https://github.com/ColuLocalNetwork/inventory-manager/pull/3#issuecomment-412507379).